app/src/main/java/org/sopt/pingle/presentation/ui/newgroup/NewGroupActivity.kt

```diff
@@ -106,21 +142,38 @@ class NewGroupActivity : BindingActivity<ActivityNewGroupBinding>(R.layout.activ
         }
     }
 
-    private fun navigateToNewGroupAnnouncement() {
+    private fun navigateToNewGroupAnnouncement(newGroupModel: NewGroupModel) {
         Intent(this, NewGroupAnnouncementActivity::class.java).apply {
+            putExtra(NEW_GROUP_CODE, newGroupModel)
             startActivity(this)
         }
+        finish()
     }
 
     private fun navigateToPreviousPage() {
         when (binding.vpNewGroup.currentItem) {
             FIRST_PAGE -> finish()
+
             else -> binding.vpNewGroup.currentItem--
         }
     }
 
+    private fun onBackPressedBtn() {
```

Comment:
```suggestion
    private fun onBackPressedCallBack() {
```
다른 액티비티에서는 이런 네이밍을 사용하셨던데 통일 시키는 게 어떨까요